packages/runtime/src/useCases/transport/messages/MessageMapper.ts

```diff
@@ -18,11 +18,25 @@ export class MessageMapper {
         };
     }
 
-    public static toMessageWithAttachmentsDTO(message: Message, attachments: File[]): MessageWithAttachmentsDTO {
+    public static async toMessageWithAttachmentsDTO(message: Message, attachments: File[]): Promise<MessageWithAttachmentsDTO> {
```

Comment:
I have strong doubts that this really should be async. Can we achieve this differently?

Reply:
As said - the pseudonymization is NOT the task of the MessageMapper. It should already be done before in the DB.